src/components/Modal/Modal.vue

```diff
@@ -643,7 +655,7 @@ $header-size: 50px;
 		transition: transform 300ms ease;
 		border-radius: var(--border-radius-large);
 		background-color: var(--color-main-background);
-		box-shadow: 0 2px 8px rgba(0, 0, 0, .33);
+		box-shadow: 0 0 40px rgba(0, 0, 0, .2);
```

Comment:
why changing the box shadow?

Reply:
to make it less harsh, with the opacity at 90% it was not a problem but after bringing that down It was too strong IMO

Reply:
Could you add screenshots of the comparison pls :)

Reply:
Before: 
![Screenshot from 2019-11-27 16-06-54](https://user-images.githubusercontent.com/26852655/69736087-1feec480-1132-11ea-934a-6d09cec7de87.png)

After:
![Screenshot from 2019-11-27 16-07-36](https://user-images.githubusercontent.com/26852655/69736088-20875b00-1132-11ea-8f9f-2c8a06aaac03.png)